PyEMD/tests/test_checks.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import unittest
+from PyEMD.significancetest import normalize
+from PyEMD.significancetest import mean_period
+from PyEMD.significancetest import energy
+from PyEMD.significancetest import significance_apriori
+from PyEMD.significancetest import significance_aposteriori
+from PyEMD.significancetest import whitenoise_check
+from PyEMD.significancetest import normalize
+
+class TestCase(unittest.TestCase):
```

Comment:
Previously I only briefly looked at tests but it might be that next time I check it'll be ready for merge so let me highlight a few things.

When writing tests, please think what you want to verify from the tested method. For example, when testing addition operator I not only want to make sure that the output has the same type as inputs but also want to make sure that it actually is adding values. Also, every method has limitations and it'll break and it's important to check that we correctly show these limitations.

Being more specific to your tests:
* Normalize test only checks for length but it actually should check that the signal is normalized. That means checking the mean value being close to zero and norm of amplitude being close to one. When using on normalized signal it should return the same signal.
* Mean period of a sinusoid is well known; you can check for more than just the type of output.
* Significance tests assume `alpha` to be within (0, 1) range. What happens outside that range? It should fail, right? How about `alpha = -10`?

Reply:
I removed the normalize function as it is not being used in `whitenoise_check` anymore. Some tests/checks are being done in the code itself using `asserts` like the value for alpha within (0,1). Also, while testing the code I realized that the code for finding mean period was not working as expected for signals having 1-3 peaks using `argrelextrema`. Found a better solution using `from scipy.signal import find_peaks`.